src/test/regress/upgrade/config.py

```diff
@@ -0,0 +1,41 @@
+OLD_BINDIR = 'OLD_BINDIR'
+NEW_BINDIR = 'NEW_BINDIR'
+TEMP_DIR = 'TEMP_DIR'
+NEW_DATADIR = 'NEW_DATADIR'
+OLD_DATADIR = 'OLD_DATADIR'
+PG_SRCDIR = 'PG_SRCDIR'
+BEFORE_UPGRADE_SCHEDULE = './before_upgrade_schedule'
+AFTER_UPGRADE_SCHEDULE = './after_upgrade_schedule'
+
+config = {
```

Comment:
I can see this dictionary works, but it is a bit fragile.
Would it be much work to use a class and members instead of a dict?